3.6/alpine/Dockerfile

```diff
@@ -58,11 +58,19 @@ RUN set -ex \
 		tk-dev \
 		xz-dev \
 		zlib-dev \
+                libffi-dev \
+                expat-dev \
 # add build deps before removing fetch deps in case there's overlap
 	&& apk del .fetch-deps \
 	\
 	&& cd /usr/src/python \
+        && rm -r Modules/expat \
+		Modules/zlib \
+		Modules/_ctypes/darwin* \
+		Modules/_ctypes/libffi* \
```

Comment:
There seems to be a mix of tabs and spaces throughout these changes; can you ensure to only use tabs. :pray: Thanks!